feature/experimental/p4rt/wbb/example/main.go

```diff
@@ -0,0 +1,375 @@
+		DeviceId:   arbMsg1.Arb.DeviceId,
+		ElectionId: arbMsg1.Arb.ElectionId,
+		Updates: wbb.AclWbbIngressTableEntryGet([]*wbb.AclWbbIngressTableEntryInfo{
+			&wbb.AclWbbIngressTableEntryInfo{
+				Type:          p4_v1.Update_INSERT,
+				EtherType:     0x6007,
+				EtherTypeMask: 0xFFFF,
+			},
+			&wbb.AclWbbIngressTableEntryInfo{
+				Type:    p4_v1.Update_INSERT,
+				IsIpv4:  0x1,
+				Ttl:     0x1,
+				TtlMask: 0xFF,
+			},
+		}),
+		Atomicity: p4_v1.WriteRequest_CONTINUE_ON_ERROR,
+	})
+	if err != nil {
+		countOK, countNotOK, errDetails := p4rt_client.P4RTWriteErrParse(err)
+		if glog.V(2) {
+			glog.Infof("Write Partial Errors %d/%d: %s", countOK, countNotOK, errDetails)
+		}
+	} else {
+		glog.Info("Write Success")
+	}
+
+	// Read ALL and log
+	rStream, rErr := client0.Read(&p4_v1.ReadRequest{
+		DeviceId: arbMsg1.Arb.DeviceId,
+		Entities: []*p4_v1.Entity{
+			&p4_v1.Entity{
+				Entity: &p4_v1.Entity_TableEntry{},
+			},
+		},
+	})
+	if rErr != nil {
+		glog.Fatal(rErr)
+	}
+	for {
+		readResp, respErr := rStream.Recv()
+		if respErr != nil {
+			glog.Warningf("Read Response Err: %s", respErr)
+			break
+		} else {
+			if glog.V(2) {
+				glog.Infof("Read Response: %s", readResp)
+			}
+		}
+	}
+
+	// Write is not associated with any streams, but we have to use the primary's
+	err = client0.Write(&p4_v1.WriteRequest{
+		DeviceId:   arbMsg1.Arb.DeviceId,
+		ElectionId: arbMsg1.Arb.ElectionId,
+		Updates: wbb.AclWbbIngressTableEntryGet([]*wbb.AclWbbIngressTableEntryInfo{
+			&wbb.AclWbbIngressTableEntryInfo{
+				Type:    p4_v1.Update_DELETE,
+				IsIpv4:  0x1,
+				Ttl:     0x1,
+				TtlMask: 0xFF,
+			},
+			&wbb.AclWbbIngressTableEntryInfo{
+				Type:    p4_v1.Update_INSERT,
+				IsIpv6:  0x1,
+				Ttl:     0x2,
+				TtlMask: 0xFF,
+			},
+		}),
+		Atomicity: p4_v1.WriteRequest_CONTINUE_ON_ERROR,
+	})
+	if err != nil {
+		countOK, countNotOK, errDetails := p4rt_client.P4RTWriteErrParse(err)
+		if glog.V(2) {
+			glog.Infof("Write Partial Errors %d/%d: %s", countOK, countNotOK, errDetails)
+		}
+	} else {
+		glog.Info("Write Success")
+	}
+
+	// Read ALL and log
+	rStream, rErr = client0.Read(&p4_v1.ReadRequest{
+		DeviceId: arbMsg1.Arb.DeviceId,
+		Entities: []*p4_v1.Entity{
+			&p4_v1.Entity{
+				Entity: &p4_v1.Entity_TableEntry{},
+			},
+		},
+	})
+	if rErr != nil {
+		glog.Fatal(rErr)
+	}
+	for {
+		readResp, respErr := rStream.Recv()
+		if respErr != nil {
+			glog.Warningf("Read Response Err: %s", respErr)
+			break
+		} else {
+			if glog.V(2) {
+				glog.Infof("Read Response: %s", readResp)
+			}
+		}
+	}
+
+	// Send L3 packet to ingress (on Primary channel)
+	err = client0.StreamChannelSendMsg(
+		&client0Stream1Name, &p4_v1.StreamMessageRequest{
+			Update: &p4_v1.StreamMessageRequest_Packet{
+				Packet: &p4_v1.PacketOut{
+					Payload: utils.PacketICMPEchoRequestGet(false,
+						net.HardwareAddr{0xFF, 0xAA, 0xFA, 0xAA, 0xFF, 0xAA},
+						net.HardwareAddr{0xBD, 0xBD, 0xBD, 0xBD, 0xBD, 0xBD},
+						net.IP{10, 0, 0, 1},
```

Comment:
Please use IPs within the range mentioned in https://github.com/openconfig/featureprofiles/blob/main/CONTRIBUTING.md

Reply:
This part is from https://github.com/openconfig/featureprofiles/pull/490. I will work with Reda to resolve this. Thanks.